content/_index/samples.md

```diff
@@ -0,0 +1,12 @@
++++
+fragment = "content"
+#disabled = true
+date = "2019-11-11"
+weight = 300
+background = "light" # can influence the text color
+particles = true
++++
+
+### Samples
```

Comment:
I'm not seeing this pop-up, did you run this locally?

If so, where does it show up?

Reply:
It’s now showing near bottom of home page above the community call section but if you’re working on an overview would make sense to be there

Reply:
Ah now I see it, that's fine for me!

Over time we could curate a list of 4-6 entries which are highlighted.